components/languageSelector/LanguageSelect.tsx

```diff
@@ -0,0 +1,32 @@
+import { twMerge } from 'tailwind-merge';
+
+import type { SelectProps } from '../form/Select';
+
+/**
+ * @description LanguageSelect component for selecting a language.
+ * @param {string} [props.className=''] - Additional classes for styling.
+ * @param {Function} [props.onChange=()=>{}] - The callback function invoked when the selection changes.
+ * @param {Array} [props.options=[]] - An array of options for the select dropdown.
+ * @param {string} props.selected - The currently selected option value.
+ */
+export default function LanguageSelect({
+  className = '',
+  onChange = () => {},
+  options = [],
+  selected
+}: SelectProps) {
```

Comment:
Why are we defining `LanguageSelect` props with `SelectProps`?

Reply:
because they both have same props